public/index.html

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="fi">
```

Comment:
I set this to fi because I believe 80+% of our users are Finnish
Not sure if there will be an actual consequence of this
Site title is also in Finnish by default